docs/api/en/math/Vector4.html

```diff
@@ -325,13 +325,13 @@ <h3>
 			[method:this setComponent]( [param:Integer index], [param:Float value] )
 		</h3>
 		<p>
-			[page:Integer index] - 0, 1 or 2.<br />
```

Comment:
Hi @Mugen87 ,

i think this line should be `[page:Integer index] - 0, 1, 2 or 3.<br />` instead.
if i'm right, should i file a PR for it?

Reply:
You are right! Fixing this in a separate PR would be great 👍 .

Reply:
cool, will do that